### PR TITLE
Integrate: Split "Analyze" section into BI vs. ML

### PR DESCRIPTION
**Pull request by amotl**
## About

Just a little refactoring to set the stage for a few content additions. The content wasn't changed much, but merely moved around.

/cc @surister

### CODE REVIEW

File: docs/integrate/visualize.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ based on Apache Superset.
 
 ```{div}
 :style: "float: right"
-[![](https://www.cluvio.com/images/logos/cluvio/cluvio-logo-full_color-on_dark.svg){w=180px}](https://www.cluvio.com/)
+[![cluvio-logo-full_color-on_dark.svg](https://github.com/crate/crate-clients-tools/assets/453543/cac142ef-412a-4a67-a63f-bf9d1ce92c84){w=180px}](https://www.cluvio.com/)
```
Comment (amotl) [inline docs/integrate/visualize.md:66]: I am just using GitHub as image host here. It worked well for a while already, so I never looked back.

/cc @matkuliak

File: docs/integrate/visualize.md [hunk 2]
```diff
@@ -78,8 +78,8 @@ your entire company in few minutes.
 
 - [Data Analysis with Cluvio and CrateDB]
 
-![](https://www.cluvio.com/images/features/custom-filters.png){h=200px}
-![](https://www.cluvio.com/images/features/report-creator.png){h=200px}
+![custom-filters.png](https://github.com/crate/crate-clients-tools/assets/453543/49ca6a35-239e-4915-951c-db6649fd35a4){h=200px}
+![report-creator.png](https://github.com/crate/crate-clients-tools/assets/453543/844a5ffd-0b92-4c77-8cdd-0b5cc5b392b1){h=200px}
```
Comment (amotl) [inline docs/integrate/visualize.md:82]: Dito.

File: docs/integrate/bi.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
+(bi)=
 (bi-tools)=
-(analyze)=
-# Analytics with CrateDB
+# Business Analytics and Intelligence with CrateDB
 
-This documentation section enumerates analytics applications and frameworks,
-which can be used for analyzing data in CrateDB.
+This documentation section enumerates business analytics applications
```
Comment (matriv) [inline docs/integrate/bi.md:5]:
```suggestion
This documentation section lists business analytics applications
```
or `contains` sounds better imho

File: docs/integrate/ml.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+(ml)=
+(ml-tools)=
+# Machine Learning with CrateDB
+
+This documentation section enumerates machine learning applications and frameworks,
```
Comment (matriv) [inline docs/integrate/ml.md:5]: same

**Review by matriv — APPROVED** [review]
LGTM, minor suggestions

File: docs/integrate/bi.md (hunk 3 above)
Comment (amotl) [reply docs/integrate/bi.md]: Thank you.

**Pull request merged**